src/aliceVision/sfmDataIO/AlembicImporter.cpp

```diff
@@ -427,6 +428,10 @@ bool readCamera(const Version & abcVersion, const ICamera& camera, const M44d& m
       {
         intrinsicLocked = getAbcProp<Alembic::Abc::IBoolProperty>(userProps, *propHeader, "mvg_intrinsicLocked", sampleFrame);
       }
+      if(const Alembic::Abc::PropertyHeader *propHeader = userProps.getPropertyHeader("mvg_intrinsicFocalRatioLocked"))
+      {
+        lockRatio = getAbcProp<Alembic::Abc::IBoolProperty>(userProps, *propHeader, "mvg_intrinsicFocalRatioLocked", sampleFrame);
+      }
```

Comment:
To be symetric with json
```suggestion
      if(const Alembic::Abc::PropertyHeader *propHeader = userProps.getPropertyHeader("mvg_intrinsicPixelRatioLock"))
      {
        lockRatio = getAbcProp<Alembic::Abc::IBoolProperty>(userProps, *propHeader, "mvg_intrinsicPixelRatioLock", sampleFrame);
      }
```